jsconfig.json

```diff
@@ -0,0 +1,124 @@
+{
+  "compilerOptions": {
+    "target": "ESNext",
+    "allowSyntheticDefaultImports": true,
+    "strictNullChecks": true,
+    "lib": [
+      "DOM",
+      "DOM.Iterable",
+      "ESNext"
+    ],
+    "module": "ESNext",
+    "moduleResolution": "NodeNext",
+    "baseUrl": ".",
+    "paths": {
```

Comment:
We don't need all these aliases in neeto-hotkeys, but since it's common in neeto, I decided to keep it as it is. Let me know if we need to remove them.

Reply:
Please clean them up & merge the PR yourself.
@anmol-kumar0815 _a